publication-commons/src/test/resources/features/NotAllowedAccessToDynamoWithoutUserCredentials.feature

```diff
@@ -0,0 +1,76 @@
+Feature:
+  DynamoDBPublicationService does not allow reading, inserting or modifying an entry without
+  the caller of the service specifying the username who is performing the action.
+
+
+  Background:
+    Given that PublicationService exists
+
+  Scenario: PublicationService requires username for reading a publication
+    Given that PublicationService provides a READ method for reading a single publication
+    And that READ method is being called by a module or an application
+    When the READ call does not include a username as a parameter
+    Then the module or the application cannot call the READ method
+
+  Scenario: PublicationService requires non empty username for reading a publication
+    Given that PublicationService provides a READ method for reading a single publication
+    And that READ method is being called by a module or an application
+    When the READ call includes a username as a parameter
+    And the parameter is null or empty
+    Then the READ method returns an error message that empty values are not allowed
+
+
+  Scenario: PublicationService requires username for listing a set of publications
+    Given that PublicationService provides a LIST method for reading a set of publications
```

Comment:
Again, if the thing. is published, it. can be listable, c.f. listing publications on public profile